File: 27-remove-element/remove-element.js
/**
 * @param {number[]} nums
 * @param {number} val
 * @return {number}
 */
var removeElement = function(nums, val) {
    let count = []
    for(let num of nums){
        if(num != val){
            count.push(num)
        }
    }
    let numslength = count.length
    for(let i =0;i<nums.length;i++){
        nums[i] = count[i]
    }
    return numslength
};

File: 27-remove-element/remove-element.py
class Solution(object):
    def removeElement(self, nums, val):
        k = 0
        while k < len(nums):
            if nums[k] == val:
                del nums[k]
            else :
                k +=1
        """
        :type nums: List[int]
        :type val: int
        :rtype: int
        """
        